source/extensions/common/wasm/context.h

```diff
@@ -53,6 +55,12 @@ struct Plugin {
 };
 using PluginSharedPtr = std::shared_ptr<Plugin>;
 
+// Opaque context object.
+class ContextObject {
```

Comment:
Rename this StoredObject and we can reuse it for storing things in the Wasm object as well.